opal/corelib/runtime.js

```diff
@@ -433,21 +439,105 @@
     @param [RubyClass] klass the target class to include module into
     @returns [null]
   */
+  function bridge() {
+    var target, donator, from, name, body, ancestors, id, methods, method, i, ancestor, bridged, length;
+
+    if (arguments.length === 4) {
+      target    = arguments[0];
+      from      = arguments[1];
+      name      = arguments[2];
+      body      = arguments[3];
+      ancestors = target.$$bridge.$ancestors();
+
+      // order important here, we have to check for method presence in
+      // ancestors from the bridged class to the last ancestor
+      for (i = 0, length = ancestors.length; i < length; i++) {
+        ancestor = ancestors[i];
+
+        if ($hasOwn.call(ancestor.$$proto, name) &&
+            ancestor.$$proto[name] &&
+            !ancestor.$$proto[name].$$donated &&
+            !ancestor.$$proto[name].$$stub &&
+            ancestor !== from) {
+          break;
+        }
+
+        if (ancestor == from) {
```

Comment:
I don't think too, it's it just raises the above question in my brain when I see it :)
If there's not particular reason to do otherwise I'd change it to `===` just because it's the default.